### ISSUE TITLE
Question about the loss function

### ISSUE THREAD

**Issue by LoveSiameseCat** [nii-yamagishilab/Capsule-Forensics-v2#9]
Hi:
  This is a nice work. I'm curious that you use cross entropy loss to replace the original loss, did you make a comparison between the two loss?

**Comment by honghuy127** [comment]
Hi, thank you.
I did a non-official comparison a long time ago but did not include it into our paper. The cross entropy loss seemed to be slightly better than the original loss but not always. You could easily verify it.

**Comment by LoveSiameseCat** [comment]
Thank you for your answer, I get it.
